app/redux/actions/files.ts

```diff
@@ -31,51 +31,6 @@ export function getFilesForPost(postId: string): ActionFunc {
     };
 }
 
-export function uploadFile(channelId: string, rootId: string, clientIds: Array<string>, fileFormData: File, formBoundary: string): ActionFunc {
```

Comment:
Post attachments don't use this, right?

99% certain, but just checking.

Reply:
nope, it doesn't